app.rb

```diff
@@ -2,7 +2,7 @@
 
 Bundler.require
 
-Dir['./lib/mcm/**/*.rb'].sort.each do |file|
+Dir['./lib/mcm/**/*.rb'].each do |file|
```

Comment:
For this and other comparable lines, what was the original purpose of the `sort` call?

Reply:
It was required to handle imports correctly in the original repo I borrowed this line from, but doesn't affect the MCM